docker-compose.yml

```diff
@@ -0,0 +1,27 @@
+version: '3'
+services:
+  web:
+    build: .
+    image: vijos/vj4
+    restart: always
+    # set arguments for vj4.server below
+    command: vj4.server --db-host=mongodb --mq-host=rabbitmq --listen=http://0.0.0.0:80
```

Comment:
`listen=http://0.0.0.0:80` might be a bad default setting.
What about unix socket?